src/soc-support.md

```diff
@@ -0,0 +1,183 @@
+# Guide for Silicon Vendors to Enable Rust Support for Their SoCs
+
+## Introduction
+
+Rust has emerged as a powerful and safety-focused programming language, gaining
+traction among embedded developers. Silicon vendors who wish to enable Rust
+support for their System-on-Chip (SoC) products can benefit from this trend by
+attracting a growing community of Rust developers.
+
+This guide aims to help silicon vendors enable Rust support, either
+independently or by empowering third-party developers. It outlines the
+essential resources, tasks, and priorities required to foster a robust Rust
+ecosystem around their System-on-Chip (SoC).
+
+**Note:** For assistance with strategy in engaging with the community, we
+recommend reaching out to the Embedded Rust Working Group (REWG) leads. They
+can provide valuable insights and support to help you navigate the process
+effectively.
+
```

Comment:
i think step 0 is missing: support for the target architecture in LLVM (and step 0b: rustc using an LLVM version which contains said support).
the alternative is the manufacturer maintaining an LLVM & rustc fork (until things get upstreamed, anyway), see what espressif is doing with `espup` for the xtensa arch.

Reply:
This would fit in a new section I am still working, which a suggested overall flow. This was an idea from @diondokter and  @jamesmunns mentioned the [How do I add support for a new microcontroller to embassy?](https://embassy.dev/book/#_how_do_i_add_support_for_a_new_microcontroller_to_embassy) he wrote in Embassy documentation.

Reply:
Added in new commit